_posts/2023-06-19-pride-spotlight-rebecca-heineman.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "US-RSE Pride Month Spotlight - Rebecca Heineman"
+tags: [dei, pride]
+author: Mattie Niznik
+---
+
+US-RSE's [DEI working group (DEI-WG)](https://us-rse.org/wg/dei/) is proud to
+help US-RSE celebrate and participate in [Pride
+Month](https://www.loc.gov/lgbt-pride-month/). Each week during Pride Month,
+the US-RSE will spotlight a member of the LGBTQIA+ community who has been
+involved in computing, science, engineering, and/or math, and have inspired our
+members through their accomplishments in their careers and their personal
+stories.
+
+## This week’s Pride Month spotlight features Rebecca Heineman
+
+{% include image.html
+url="https://upload.wikimedia.org/wikipedia/commons/5/52/Rebecca_Heineman_on_Burgertime.jpg"
+description="Rebecca Heineman, CC BY 3.0 &lt;https://creativecommons.org/licenses/by/3.0&gt;, via Wikimedia Commons"
```

Comment:
The imbedded url here and the next line are giving the URL checker a hard time

Reply:
any idea why? it resolves fine for me

Reply:
We did have something similar (I honestly thought it was the same) in a previous post. I guess I can remove it if need be, but I'll see if changing it a bit helps.

https://github.com/USRSE/usrse.github.io/blob/522089130ea30632734b43369f16569c283a248c/_posts/2023-06-05-pride-spotlight-sophie-wilson.md?plain=1#L18-L22

Reply:
@peterfpeterson @kescobo Looks like this is fixed (at least it builds and URL check runs without error).